### ISSUE TITLE
CapsNet model on CIFAR10(Quick CIFAR10 ensemble test results for capsule)

### ISSUE THREAD

**Issue by UCASJingKun** [tensorflow/models#3536]
### Describe the problem
ResourceExhaustedError (see above for traceback): OOM when allocating tensor with shape[921600,160] and type float on /job:localhost/replica:0/task:0/device:GPU:0 by allocator GPU_0_bfc
         [[Node: tower_0/capsule2/Wx_plus_b/Sum = Sum[T=DT_FLOAT, Tidx=DT_INT32, keep_dims=false, _device="/job:localhost/replica:0/task:0/device:GPU:0"](tower_0/capsule2/Wx_plus_b/mul, tower_0/capsule2/routing/TensorArrayReadV3/index)]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

         [[Node: tower_0/capsule2/routing/while/Reshape/_73 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_189_tower_0/capsule2/routing/while/Reshape", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](^_clooptower_0/capsule2/routing/while/Softmax/_45)]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

### Source code / logs
models/research/capsules/experment.py

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by wt-huang** [comment]
Closing this issue due to inactivity, feel free to reopen if this is still an issue.
